api/v1beta1/elfcluster_types.go

```diff
@@ -39,6 +39,12 @@ type ElfClusterSpec struct {
 	// ControlPlaneEndpoint represents the endpoint used to communicate with the control plane.
 	// +optional
 	ControlPlaneEndpoint APIEndpoint `json:"controlPlaneEndpoint"`
+
+	// GraciedShutdown is the config for whether cluster machine should grace shutdown
```

Comment:
// VMGracefulShutdown indicates the VMs in this ElfCluster should shutdown gracefully when deleting the VMs.
	// Default to false because sometimes the OS stuck when shutting down gracefully.
	// +optional